src/scripts/components/flash.js

```diff
@@ -0,0 +1,123 @@
+/*
+  handles showing the floating notifications
+
+  usage:
+    new api.flash('type', 'message');
```

Comment:
`api`?

Reply:
Ah i see, this file is a copy-paste from the template. In this case i think it's fine, and we can refactor this js module later. I won't add any comments to this part then 👍